### PR TITLE
feat(19341): inactive public servant role

### PR DESCRIPTION
**Pull request by nnorbert**
inactive public servant role

### CODE REVIEW

File: apps/payments/app/[locale]/(hosted)/inactivePublicServant/page.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { getTranslations } from "next-intl/server";
+
+export default async () => {
+  const t = await getTranslations("InactivePublicServant");
```
Comment (alfonsograziano) [inline apps/payments/app/[locale]/(hosted)/inactivePublicServant/page.tsx:4]: Please let's keep in mind that we need the same in Messaging

**Review by alfonsograziano — DISMISSED** [review]
LGTM

**Review by peschina — DISMISSED** [review]
LGTM!

File: apps/messaging/app/[locale]/inactivePublicServant/page.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import "design-system/dist/style.css";
+import "design-system/dist/esm/index.css";
+import Header from "../Header";
+import Footer from "../Footer";
+import { getTranslations } from "next-intl/server";
+import type { Metadata } from "next";
+import { getLinks } from "../../utils/messaging";
+
+export const metadata: Metadata = {
```
Comment (peschina) [inline apps/messaging/app/[locale]/inactivePublicServant/page.tsx:9]: Do we need this? Put here the title would be set for this page only

File: apps/messaging/app/[locale]/page.tsx [hunk 3]
```diff
@@ -2,6 +2,14 @@ import { redirect, RedirectType } from "next/navigation";
 import { AuthenticationFactory } from "../utils/authentication-factory";
 
 export default async () => {
+  console.log(await AuthenticationFactory.getInstance().getContext());
```
Comment (peschina) [inline apps/messaging/app/[locale]/page.tsx:5]: can we remove this log?

**Review by peschina — DISMISSED** [review]
LGTM, a few minor comments

File: apps/messaging/app/[locale]/inactivePublicServant/page.tsx (hunk 2 above)
Comment (nnorbert) [reply apps/messaging/app/[locale]/inactivePublicServant/page.tsx:9]: Oh, maybe not, that is a leftover, I think.

File: apps/messaging/app/[locale]/page.tsx (hunk 3 above)
Comment (nnorbert) [reply apps/messaging/app/[locale]/page.tsx:5]: I think, we can :D

File: apps/messaging/app/[locale]/inactivePublicServant/page.tsx (hunk 2 above)
Comment (nnorbert) [reply apps/messaging/app/[locale]/inactivePublicServant/page.tsx:9]: No, this is important, without this, the page won't have any title.

File: apps/messaging/app/[locale]/page.tsx (hunk 3 above)
Comment (nnorbert) [reply apps/messaging/app/[locale]/page.tsx]: Done

File: apps/messaging/app/[locale]/inactivePublicServant/page.tsx (hunk 2 above)
Comment (peschina) [reply apps/messaging/app/[locale]/inactivePublicServant/page.tsx:9]: The page title should be set for the whole app I think, not on a single page basis. We can leave it like this, I'll address this as part of the favicon fix :)

File: apps/payments/app/[locale]/(hosted)/inactivePublicServant/page.tsx (hunk 1 above)
Comment (nnorbert) [reply apps/payments/app/[locale]/(hosted)/inactivePublicServant/page.tsx:4]: Done

**Pull request merged**